pkg/security/ebpf/probes/mprotect.go

```diff
@@ -9,17 +9,16 @@ package probes
 
 import manager "github.com/DataDog/ebpf-manager"
 
-// mprotectProbes holds the list of probes used to track mprotect events
```

Comment:
I'd like to keep the comments

Reply:
I feel like this kind of comments are the worst because they bring no value, but I can try to add some of those back

Reply:
yea, true true, i'm kinda on the fence too